src/pages/search/components/FilteredList.tsx

```diff
@@ -0,0 +1,73 @@
+import tw, { styled } from 'twin.macro'
+import FilteredUser from './FilteredUser'
+import FilteredPost from './FilteredPost'
+import NoResult from './NoResult'
+
+interface FilteredListProps {
+  type: 'user' | 'all'
+  users?: User[]
+  posts?: Post[]
+  isSearched?: boolean
+}
+
+const Container = styled.div`
+  ${tw`flex flex-col gap-3 grow`}
+`
+
+const HorizontalSlide = styled.div`
+  ${tw`pb-3 flex items-center gap-3 overflow-x-scroll`}
+`
+
+const GridPostSection = styled.div`
+  ${tw`grid grid-cols-2 gap-2`}
+`
+
+const FilteredList = ({
+  type,
+  users,
+  posts,
+  isSearched
+}: FilteredListProps) => {
+  return (
+    <Container>
+      {type === 'user' &&
+        users?.map((user) => (
+          <FilteredUser
+            image={user.image}
+            fullName={user.fullName}
+            email={user.email}
+          />
```

Comment:
P5 : 저는 객체 분해할당 했을 때 2개 이상 나눠진다면 분해할당을 많이 활용하는 편인데, 이러면 조금 더 간결해지는거 같은 느낌이 들더라구요. 이거는 스타일차이이니 적용안하셔도 될 거 같습니다. 
```suggestion
  users?.map(({image,fullName,email) => (
          <FilteredUser
            image={image}
            fullName={fullName}
            email={email}
          />

Reply:
👍